sdk/core/Azure.Core.TestFramework/src/ClientTestFixtureAttribute.cs

```diff
@@ -15,16 +15,36 @@ public class ClientTestFixtureAttribute : NUnitAttribute, IFixtureBuilder2, IPre
     {
         public static readonly string SyncOnlyKey = "SyncOnly";
 
+        private readonly object[] _additionalParameters;
         private readonly object[] _serviceVersions;
         private readonly int? _maxServiceVersion;
 
+        /// <summary>
+        /// Initializes an instance of the <see cref="ClientTestFixtureAttribute"/> accepting additional fixture parameters.
+        /// </summary>
+        /// <param name="serviceVersions">The set of service versions that will be passed to the test suite.</param>
         public ClientTestFixtureAttribute(params object[] serviceVersions)
```

Comment:
call the other ctor.

Reply:
because of the `params` it won't work as expected (unless there is a syntax trick I am missing)

Reply:
https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQBMQGoA+ABATARgFgAobAZgAI8KBhAGwEs4A7GAFTlgDEGAPGAK4I4AQRgwEDYAPgUA3iQpKKi5eVqMW7TjB78ho8ZOnwAFAAcICCGCgUA9sABWcAMYwA2gF0KURADcGVzgANUQoBntmKABKEAoYAAsGKFM/BEDgsIQIqKgkChEEawBPADoAUTBzGBKAHkcXdwA+UxiY+QBfVSV1eiZWDm4+QWExCSkZOFNGt08fdMzQ8Mjogtn3bwoIFBQGGFWIOgAFKxs4eByOuW7iEk6gA==

Reply:
It compiles, but NUnit doesn't find it the same way anymore for some reason.

Reply:
I'm very confused as to why that can be considering I don't even think that adding `:this()` is observable via reflection..

Reply:
ugh - it was a bug in my ctor - it works as expected,,